vizro-core/docs/pages/user-guides/aggrid.md

```diff
@@ -0,0 +1,281 @@
+# How to use AGGrids
+
+This guide shows you how to use the [AG Grid](https://www.ag-grid.com/) to visualize your tabular data in the dashboard.
```

Comment:
I'm still deciding whether we should use all US English or UK English spelling, so am going to leave this for now, but it could change (when I make the edits) to "visualise" at some point...